Fails to build on arm64 due to wrong pointer type

Not sure why, but it seems like something in gdal_sys expects a different pointer type for ESRI when compiling for arm64. This is the only error I get:

```rust
error[E0308]: mismatched types
   --> /app/.cargo/registry/src/github.com-1ecc6299db9ec823/gdal-0.6.0/src/spatial_ref/srs.rs:157:62
    |
157 |         let rv = unsafe { gdal_sys::OSRImportFromESRI(c_obj, ptrs.as_mut_ptr()) };
    |                                                              ^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*mut *mut u8`
               found raw pointer `*mut *mut i8`
```

I'm going to dig further into this.

--- a/src/spatial_ref/srs.rs
+++ b/src/spatial_ref/srs.rs
@@ -1,6 +1,6 @@
 use crate::utils::{_last_cpl_err, _last_null_pointer_err, _string};
 use gdal_sys::{self, CPLErr, OGRCoordinateTransformationH, OGRErr, OGRSpatialReferenceH};
-use libc::c_int;
+use libc::{c_int, c_char};
 use std::ffi::{CStr, CString};
 use std::ptr;
 use std::str::FromStr;
@@ -166,7 +166,7 @@ impl SpatialRef {
 
     pub fn from_esri(esri_wkt: &str) -> Result<SpatialRef> {
         let c_str = CString::new(esri_wkt)?;
-        let mut ptrs = vec![c_str.as_ptr() as *mut i8, ptr::null_mut()];
+        let mut ptrs = vec![c_str.as_ptr() as *mut c_char, ptr::null_mut()];
         let null_ptr = ptr::null_mut();
         let c_obj = unsafe { gdal_sys::OSRNewSpatialReference(null_ptr) };
         let rv = unsafe { gdal_sys::OSRImportFromESRI(c_obj, ptrs.as_mut_ptr()) };

